src/Components/Footer.js

```diff
@@ -1,34 +1,34 @@
 import React from 'react';
-
+import {Link ,Navigate,useNavigate} from 'react-router-dom';
```

Comment:
Hey, please remove `Navigate & useNavigate` here, as they are unused variables.
```suggestion
import {Link} from 'react-router-dom';
```